Cargo.toml

```diff
@@ -122,6 +122,7 @@ tracing-subscriber = { version = "=0.3.18", features = ["env-filter", "json"] }
 typomania = { version = "=0.1.2", default-features = false }
 url = "=2.5.4"
 unicode-xid = "=0.2.6"
+krata-tokio-tar = "=0.4.2"
```

Comment:
I assume this is a fork and there is a reason for not using the original?

Reply:
Since the original `tokio-tar` seems unmaintained, and I know it's used by `uv` and has been reviewed!